pkg/controllers/disruption/controller.go

```diff
@@ -58,8 +58,13 @@ type Controller struct {
 	lastRun       map[string]time.Time
 }
 
-// pollingPeriod that we inspect cluster to look for opportunities to disrupt
-const pollingPeriod = 10 * time.Second
+const (
+	// pollingPeriod that we inspect cluster to look for opportunities to disrupt
+	pollingPeriod = 10 * time.Second
+
+	// commandExecutionDelay is the duration spent waiting after tainting candidate nodes before validaing the command
+	commandValidationDelay = 5 * time.Second
```

Comment:
Is this a dart throw? Or are there metrics you're using to drive this decision

Reply:
Resolving, updated approach does not involve taint and wait strategy. This number was a slight dart through that was justified by the default value for the same value in CAS.